sdk/storage/azure-storage-blob/tests/test_common_blob.py

```diff
@@ -48,6 +48,7 @@
     generate_container_sas,
     upload_blob_to_url)
 from azure.storage.blob._generated.models import RehydratePriority
+from azure.storage.blob._shared.models import Services
```

Comment:
Sorry, I forgot to mention this before, can you import this from `azure.storage.blob` instead of the private import? It is an exported class so we should import from the main package. Same for async.

Reply:
Is it exported?
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-blob/azure/storage/blob/__init__.py

Reply:
Ah, no its probably not since we didn't expose the option before! We should export it from each package then.